specs.html

```diff
@@ -26,106 +26,65 @@
   <ol>
     <li>
       <p>
-        <a href="https://help.github.com/articles/creating-a-new-repository">Create a new GitHub
-        repository</a>. The spec's short name in lowercase is usually a good choice.
+        <a href="https://labs.w3.org/hatchery/repo-manager/repo/new">Create a new GitHub repository</a> using
+        <a href="https://labs.w3.org/hatchery/repo-manager/">ash-nazg</a>.
+        The spec's <em>shortname</em> in lowercase is usually a good choice.
       </p>
     </li>
     <li>
       <p>
-        Clone the repository locally,
+        Once the new repository is created, find it on GitHub and set up
+        <a href="https://github.com/blog/2228-simpler-github-pages-publishing">GitHub Pages</a>
+        to use the default branch, <code>master</code> (<em>Settings</em> / <em>Options</em> / <em>GitHub Pages</em>).
```

Comment:
Never mind, I see that's what is recommended in the [blog post](https://github.com/blog/2228-simpler-github-pages-publishing) mentioned in the FAQ.